### ISSUE TITLE
[기능] 학습 코드에 새로운 loss function 추가

### ISSUE THREAD

**Issue by Seongho-Cho12** [dandylionkiddo/guideWay#5]
지금의 cross-entropy loss로는 데이터 불균형 시 데이터가 많은 background 클래스에 집중하여 학습하는 경향이 있습니다.

대신 focal loss, dice loss를 학습에 사용하면 데이터 불균형으로 인한 학습 문제가 해결될 수 있을 것으로 보입니다.
